installer/manifests/keptn/templates/api-gateway-nginx.yaml

```diff
@@ -383,6 +383,11 @@ spec:
     spec:
       {{- include "keptn.apiGatewayNginx.pod-security-context" . | nindent 6 }}
       terminationGracePeriodSeconds: {{ .Values.apiGatewayNginx.gracePeriod | default 60 }}
+      initContainers:
+        {{- include "keptn.initContainers.wait-for-nats" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-keptn-mongo" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-mongodb-datastore" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-shipyard-controller" . | nindent 8 }}
```

Comment:
I don't think this is necessary for nginx since it relies already on the K8s services to be ready